### DISCUSSION TITLE
What is the typescript's @types/mongoose version for mongoose 5.13.14

### DISCUSSION THREAD

**Discussion by jamesdeath123** [Automattic/mongoose#11755] [Q&A]
My project is using mongoose and typescript. I think back in 5.13.14, we still have to import and the types explicitly. But I cannot find any documentation that describes which latest `@types/mongoose` version I can use to make `5.13.14` works. Could anyone point me to the right version?

**Comment by vkarpov15** [comment]
There isn't one. Mongoose 5.13 has its own TypeScript types.
